### DISCUSSION TITLE
Product Addon Suggestion

### DISCUSSION THREAD

**Discussion by magedfathi** [Blair2004/NexoPOS#417] [Ideas]
i would like to suggest the ADDON feature like

Sandwich ADDON Category
Cheese Addon
Extra Frech Fries

and while creating the product i can choose the ADDON CAtegories so it will appear after choosing the product from the POS

**Comment by Blair2004** [comment]
This is already supported by modifiers on Gastro 4.x

https://my.nexopos.com/en/marketplace/item/gastro-4x-restaurant-extension-for-nexopos-4x

**Reply by magedfathi** [reply]
Noted
